cli/ioctl/cmd/config/configsetget.go

```diff
@@ -110,6 +137,13 @@ func set(args []string) (string, error) {
 		ReadConfig.SecureConnect = !Insecure
 	case "wallet":
 		ReadConfig.Wallet = args[1]
+	case "currentcontext":
+		err1 := validator.ValidateAlias(args[1])
```

Comment:
```suggestion
		if validator.ValidateAlias(args[1]) != nil &&  validator.ValidateAddress(args[1]) != nil {
```

Short circuit

Reply:
need check both,input could be alias or address,short circuit only check one
if input is address,ValidateAlias will return error,that's not right.
I need ValidateAddress to check if this address is right,only both ValidateAlias and ValidateAddress return error will return error